lib/ansible/modules/cloud/kubevirt/kubevirt_vm.py

```diff
@@ -29,10 +29,11 @@
     state:
         description:
             - Set the virtual machine to either I(present), I(absent), I(running) or I(stopped).
-            - "I(present) - Create or update virtual machine."
-            - "I(absent) - Removes virtual machine."
-            - "I(running) - Create or update virtual machine and run it."
-            - "I(stopped) - Stops the virtual machine."
+            - "Ephemeral virtual machines do not support I(running)/I(stopped) states."
+            - "I(present) - Create or update a virtual machine."
+            - "I(absent) - Remove a virtual machine."
+            - "I(running) - Create or update a non-ephemeral virtual machine and run it."
+            - "I(stopped) - Stop a non-ephemeral virtual machine."
```

Comment:
Not sure if there is use case for it , but there may be use case that user want to ensure VM is created and stopped, I would still prefer to support it, if it used to work,.